backend/src/components/public.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
```

Comment:
While I love the idea of having a public route to make it clear what functionality is being exposed publicly I think the components can still be aligned along functional/business concerns. While it's not exactly "lookup" data it does do caching you could certainly lump this into lookup.js or rename this component to be just for system messages.

Reply:
it's updated